### PR TITLE
sqlalchemy: Added requirements.txt and changed headings in output.txt

### PR DESCRIPTION
**Pull request by shravan97**
## Description of new Instant Answer, or changes

Added requirements.txt file to sqlalchemy fathead. Also, the ``` <span> ``` tags are changed to ``` <pre><code> ```  for headings in output.txt . 

## Related Issues and Discussions

#462 

## People to notify

@moollaza 

## Testing & Review
To be completed by Language Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [x] Title follows correct format (Specifies Instant Answer + Purpose)
- [x] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [x] Instant Answer page is correctly filled out and contains:
    - One topic for the Search Space Language (Java, Python, Scala, Ruby, etc.)
    - One topic from: Reference, Help, Libraries, Tools
        - Documentation Fatheads are considered "Reference"
    - Description, Data source, and 2+ example queries
    - Perl Module (e.g. "DDG::Fathead::PerlDoc" -- we only need a name, not an actual file)
    - Source Name (for "More at <source_name>" link)
    - Source Domain (must contain http:// or https:// -- can be the same as Data Source)
    - Source Info (used as Subtitle for each Article -- usually matches the IA Name)
    - 'Skip Abstract' is checked off
    - Source ID (ping @moollaza to assign one, once Fathead is ready for Beta deploy)

**Code**
- [x] Uniformly indented, well commented
- [ ] Fetch.sh and Parse.sh run without errors
- [x] Output contains no blank lines, or multi-line entries
- [ ] Fathead Tests are passing (run `$ duckpan test <fathead_id>`)
    - Tester should report any failures

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/sqlalchemy

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/fathead/sqlalchemy/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+bs4
```
Comment (moollaza) [inline lib/fathead/sqlalchemy/requirements.txt]: I think this needs to declare a version?

You should use `pip freeze > requirements.txt` to generate the file

File: lib/fathead/sqlalchemy/output.txt [hunk 2]
```diff
@@ -1,405 +1,405 @@
...
+items	A										<section class="prog__container"><pre><code>items() &#8594; list of D's (key, value) pairs, as 2-tuples</code></pre><p>\ninherited from the items() method of dict</p></section>	http://docs.sqlalchemy.org/en/latest/orm/internals.html#sqlalchemy.orm.instrumentation.ClassManager.items
+attributes.QueryableAttribute.match	A										<section class="prog__container"><pre><code>match(other, **kwargs)</code></pre><p>\ninherited from the match() method of ColumnOperatorsImplements a database-specific &#8216;match&#8217; operator.match() attempts to resolve to\na MATCH-like function or operator provided by the backend.\nExamples include:\nPostgreSQL - renders x @@ to_tsquery(y)\nMySQL - renders MATCH (x) AGAINST (y IN BOOLEAN MODE)\nOracle - renders CONTAINS(x, y)\nother backends may provide special implementations.\nBackends without any special implementation will emit\nthe operator as &#8220;MATCH&#8221;.  This is compatible with SQlite, for\nexample.\n</p></section>	http://docs.sqlalchemy.org/en/latest/orm/internals.html#sqlalchemy.orm.attributes.QueryableAttribute.match
+InspectionAttr	A										<section class="prog__container"><pre><code
```
Comment (moollaza) [inline lib/fathead/sqlalchemy/output.txt]: @shravan97 looking at this close, I see that you're converting a lot of different kind of markup into plain text, and the result here is pretty hard to read.

If you pull out `<p>` and `<ul>` from the source and escape their contents, you should be able to preserve the same layout we see in the documentation.

As well, I think you should be parsing the `Parameters` separately. into a `<ul>` with a proper subtitle, or ignoring it for now...

Comment (moollaza) [inline lib/fathead/sqlalchemy/output.txt]: As well, it looks like you're missing a newline and a code block in the first sentence. The result should look more like:

```
Bases: <code>sqlalchemy.event.base.Events</code>

Define events specific to mappings.
```

But we're showing:

```
Bases: sqlalchemy.event.base.EventsDefine events specific to mappings
```

**Comment by moollaza** [comment]
@shravan97 Thanks! This LGTM :+1:

Great work 🚀 

Can you please remove the `/backup` dir from git though? There's no need for us to maintain that in version control.

**Comment by moollaza** [comment]
@shravan97 one other thing I noticed -- It looks we don't have spaces after the periods in our abstracts -- any chance you can fix that?

**Pull request merged**